corehq/motech/repeaters/models.py

```diff
@@ -210,10 +216,13 @@ class SQLRepeater(models.Model):
         on_delete=models.PROTECT
     )
 
+    couch_id = models.CharField(max_length=126, null=True)
+
     class Meta:
         indexes = [
             models.Index(fields=['domain']),
             models.Index(fields=['repeater_id']),
+            models.Index(fields=['couch_id']),
```

Comment:
nit: these can all be changed to `db_index=True` in the field constructor.
`couch_id = models.CharField(max_length=126, null=True, db_index=True)`, which I think makes it slightly easier to read, and using this `Meta` only for multicolumn indices since those cannot otherwise be defined.